example/windows/flutter_embedder_example.cpp

```diff
@@ -46,7 +46,16 @@ std::string GetExecutableDirectory() {
 
 }  // namespace
 
-int main(int argc, char **argv) {
+int APIENTRY wWinMain(HINSTANCE instance,
+                      HINSTANCE prev,
+                      wchar_t *command_line,
+                      int show_command) {
+  // attach to console of 'flutter run'
```

Comment:
Nit: s/attach/Attach/, and add missing period at the end. The style guide requires that comments be punctuated as sentences.

Reply:
> I don't currently have access to my Windows machine. Did you confirm that this version works correctly with `flutter run`, attaching to the observatory and accepting/processing key input to trigger things like hot reload?

yes, I'm sure. It works fine in both scenarios (flutter run or VS debugger).
![WXWorkCapture_15671489902245](https://user-images.githubusercontent.com/1241023/64000500-7ac74e80-cb38-11e9-8064-fd74982c8541.png)
![WXWorkCapture_1567149044653](https://user-images.githubusercontent.com/1241023/64000507-7f8c0280-cb38-11e9-912d-41af6fd9ac47.png)

Reply:
> Nit: s/attach/Attach/, and add missing period at the end. The style guide requires that comments be punctuated as sentences.

DONE.